kubespawner/spawner.py

```diff
@@ -1934,6 +1935,8 @@ def _get_pod_url(self, pod):
         else:
             proto = "http"
             hostname = pod["status"]["podIP"]
+            if type(ipaddress.ip_address(hostname)) == ipaddress.IPv6Address:
```

Comment:
let's use a standard Python type check:

```suggestion
            if isinstance(ipaddress.ip_address(hostname), ipaddress.IPv6Address):
```